code/modules/projectiles/projectile.dm

```diff
@@ -621,13 +621,15 @@
 			return FALSE
 		if(HAS_TRAIT(living_target, TRAIT_IMMOBILIZED) && HAS_TRAIT(living_target, TRAIT_FLOORED) && HAS_TRAIT(living_target, TRAIT_HANDS_BLOCKED))
 			return FALSE
-		if(!hit_prone_targets)
+		if(hit_prone_targets)
 			var/mob/living/buckled_to = living_target.lowest_buckled_mob()
-			if(!buckled_to.density) // Will just be us if we're not buckled to another mob
-				return FALSE
-			if(living_target.body_position != LYING_DOWN)
+			if((istype(src, /obj/projectile/bullet/p50)) || (istype(src, /obj/projectile/bullet/pellet/stingball))) // doesn't apply to sniper rifles and ricocheted stingballs
```

Comment:
I don't like this line. There might be a better option out there. However, I can't think of any alternative.

Reply:
Add a new var to bullets which describes whether they have this behaviour or not

Reply:
thanks!